client/themes/bcgov/_bootstrap.scss

```diff
@@ -1,5 +1,5 @@
-$bcgov-font-path: "../../../node_modules/@bcgov/bootstrap-theme/dist/fonts/";
-@import "../../../node_modules/@bcgov/bootstrap-theme/dist/scss/bootstrap-theme";
+// $bcgov-font-path: "../../../node_modules/@bcgov/bootstrap-theme/dist/fonts/";
+@import "/node_modules/@bcgov/bootstrap-theme/dist/scss/bootstrap-theme";
 
-$fa-font-path: "../../../node_modules/font-awesome/fonts/";
-@import "../../../node_modules/font-awesome/scss/font-awesome";
+// $fa-font-path: "../../../node_modules/font-awesome/fonts/";
```

Comment:
What is the effect - if any - of removing these imports? Do displayed font styles change?

Reply:
Based on local tests, comparing to what's currently deployed, the fonts look like they are loading correctly. Icons are also displaying properly.

## Angular Update
![Local](https://user-images.githubusercontent.com/5605464/100479023-de288f00-30a1-11eb-9430-e834730f89f8.png)

## Current OrgBook deployment
![Production](https://user-images.githubusercontent.com/5605464/100479016-db2d9e80-30a1-11eb-82f6-36f6ed9dbd9d.png)

Reply:
Interesting, I wonder what those SASS variables do then 🤔 
I guess tests in `dev` may tell us more at this point.